src/Symfony/Component/DependencyInjection/Tests/Compiler/ResolveBindingsPassTest.php

```diff
@@ -134,4 +137,20 @@ public function testWithNonExistingSetterAndBinding()
         $pass = new ResolveBindingsPass();
         $pass->process($container);
     }
+
+    public function testSyntheticServiceWithDefaultBinds()
+    {
+        $container = new ContainerBuilder();
+        $loader = new XmlFileLoader($container, new FileLocator(realpath(__DIR__.'/../Fixtures/xml')));
+        $loader->load('services_synthetic.xml');
```

Comment:
instead of using a loader, the test case should create the definitions inline, as other ones do

Reply:
i couldn't find an example on how to define _defaults $bind in a test case

Reply:
Actually you don't need to use defaults: adding a binding on the definition runs into the issue too.
See `Definition->setBindings()`

Reply:
ok, changed test